v2/orders/order-workflows-groups.rst

```diff
@@ -0,0 +1,14 @@
+Order Workflow Groups
```

Comment:
We can remove this, it's general state_machine documentation, and orders only use the group provided by core. It's very internal.